### PR TITLE
SEO-173296 WPF Chart Technical Indicators

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @MallikaRK ,

Have renamed image folder, changed and image alt.

Regards,
Yvone.

### CODE REVIEW

File: wpf/Charts/Technical-Indicators.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ chart.TechnicalIndicators.Add(indicator);
 
 {% endtabs %}
 
-![AccumulationDistribution Indicator in WPF Chart](Technical-Indicators_images/wpf-chart-accumulation-indicator.png)
+![Accumulation distribution indicator in WPF Chart](Technical-Indicators_images/wpf-chart-accumulation-indicator.png)
```
Comment (MallikaRK) [inline wpf/Charts/Technical-Indicators.md:124]: Image FOLDER Name still in Capital. recheck

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: wpf/Charts/Technical-Indicators.md (hunk 1 above)
Comment (Yvone-Atieno) [reply wpf/Charts/Technical-Indicators.md:124]: rechecked

File: wpf/Charts/Technical-Indicators.md [hunk 2]
```diff
@@ -121,7 +121,7 @@ chart.TechnicalIndicators.Add(indicator);
 
 {% endtabs %}
 
-![AccumulationDistribution Indicator in WPF Chart](Technical-Indicators_images/wpf-chart-accumulation-indicator.png)
+![Accumulation distribution indicator in WPF Chart](technical-Indicators_images/wpf-chart-accumulation-indicator.png)
```
Comment (MallikaRK) [inline wpf/Charts/Technical-Indicators.md:124]: Still captital case exists in Image Folder Name. Check and correct it.

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

**Review by MallikaRK — APPROVED** [review]
Approved

File: wpf/Charts/Technical-Indicators.md [hunk 3]
```diff
@@ -121,7 +121,7 @@ chart.TechnicalIndicators.Add(indicator);
 
 {% endtabs %}
 
-![AccumulationDistribution Indicator in WPF Chart](Technical-Indicators_images/wpf-chart-accumulation-indicator.png)
+![Accumulation distribution indicator in WPF Chart](technical-indicators_images/wpf-chart-accumulation-indicator.png)
```
Comment (Girija-Rajendran) [inline wpf/Charts/Technical-Indicators.md:124]: Changed this name in folder

Comment (Yvone-Atieno) [reply wpf/Charts/Technical-Indicators.md:124]: the folder is changed

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**